pulsar-broker/src/test/java/org/apache/pulsar/broker/intercept/CounterBrokerInterceptor.java

```diff
@@ -66,42 +66,52 @@
     }
 
     @Override
-    public void onConnectionCreated(ServerCnx cnx){
-        log.info("Connection created {}", cnx);
+    public void onConnectionCreated(ServerCnx cnx) {
+        if (log.isDebugEnabled()) {
+            log.info("Connection created {}", cnx);
```

Comment:
Shouldn't we also change the actual log lines to match their log guard?
```suggestion
            log.debug("Connection created {}", cnx);
```